### ISSUE TITLE
Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience.

### ISSUE THREAD

**Issue by ukkpower** [miguel-perez/smoothState.js#132]
I have implemented this onto my website http://hotelieracademy.com and in Chrome I get this error when the first page loads 

```
Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check http://xhr.spec.whatwg.org/.
```

Then I get this for every click afterwards

```
Uncaught TypeError: undefined is not a function
```

I get a slightly different error is Safari.

I load one .js file at the bottom of the page

  <script src="/js/main.min.js" async="true" defer="defer"></script>

Inside I have:
-  jQuery
- smoothstate
- $('#main').smoothState({ pageCacheSize: 4, prefetch: true });
- other plugins

**Comment by ukkpower** [comment]
Another symptom I am seeing is when I hover a link sometimes it fetches a page 2-6 times.
![screen shot 2015-03-05 at 18 19 26](https://cloud.githubusercontent.com/assets/1074922/6511447/eb58be3a-c364-11e4-8984-537131f04ca6.png)

**Comment by ukkpower** [comment]
I have noticed that every link click loads the new page and the images are fetched but I also notice the main.min.js gets fetched again with the addition of ?_=7878739879

main.min.js?_=1425762489619

I have nothing on the server that should do that. Is this part of smoothState.js?

**Comment by ukkpower** [comment]
I found that it was jQuery adding the timestamp. I have copied the javascript code directly from the smoothState site.  Strange that my site experienced jQuery adding this. The problem still remains even though the timestamp is now being removed. The main.min.js seems to be loaded a second time and running twice on the page. I thought with the timestamp gone it will see that is already loaded. But it still loads anyway from cache.

```
$.ajaxPrefilter(function( options, originalOptions, jqXHR ) {
  if ( options.dataType == 'script' || originalOptions.dataType == 'script' ) {
      options.cache = true;
  }
});
```
